src/univalue/univalue.cpp

```diff
@@ -203,9 +210,95 @@ const char *uvTypeName(UniValue::VType t)
     case UniValue::VARR: return "array";
     case UniValue::VSTR: return "string";
     case UniValue::VNUM: return "number";
+    case UniValue::VREAL: return "number";
     }
 
     // not reached
     return NULL;
 }
 
+const UniValue& find_value( const UniValue& obj, const std::string& name)
+{
+    for (unsigned int i = 0; i < obj.keys.size(); i++)
```

Comment:
This is still present in [latest upstream](https://github.com/jgarzik/univalue/blob/master/lib/univalue.cpp#L279), so should be turned into a PR there.